docs/_docs/upgrading/0-to-2.md

```diff
@@ -1,140 +0,0 @@
----
-layout: docs
-title: Upgrading from 0.x to 2.x
-permalink: /docs/upgrading/0-to-2/
----
-
-Upgrading from an older version of Jekyll? A few things have changed in 1.0
```

Comment:
There aren't any hooks, but we could poll: http://guides.rubygems.org/rubygems-org-api/#gem-version-methods